src/jit/hwintrinsiclistArm64.h

```diff
@@ -41,24 +41,24 @@ HARDWARE_INTRINSIC(NI_ARM64_NONE_MOV,             None,     None,
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsByte,       Base,     AsByte,                         UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsInt16,      Base,     AsInt16,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsInt32,      Base,     AsInt32,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsSByte,      Base,     AsSByte,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsSingle,     Base,     AsSingle,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsUInt16,     Base,     AsUInt16,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector64_AsUInt32,     Base,     AsUInt32,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_As,          Base,     As,                             UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsByte,      Base,     AsByte,                         UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsDouble,    Base,     AsDouble,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsInt16,     Base,     AsInt16,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsInt32,     Base,     AsInt32,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsInt64,     Base,     AsInt64,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsSByte,     Base,     AsSByte,                        UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsSingle,    Base,     AsSingle,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsUInt16,    Base,     AsUInt16,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsUInt32,    Base,     AsUInt32,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
-HARDWARE_INTRINSIC(NI_Base_Vector128_AsUInt64,    Base,     AsUInt64,                       UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
+HARDWARE_INTRINSIC(NI_Vector64_AsByte,        Vector64,     AsByte,                         UnaryOp,       INS_invalid, INS_invalid, INS_invalid,  None )
```

Comment:
Minor point; the `Vector64` field would normally be left-aligned with `Base` above.